cmd/print.go

```diff
@@ -219,6 +258,24 @@ func (pr *PrintRunner) initializeNamespaceFilter() error {
 	return nil
 }
 
+// 1. When --output-file receives no argument, initializeOutputFile will do nothing.
+// 2. When --output-file receives an argument, initializeOutputFile will validate whether the value is correct.
+func (pr *PrintRunner) initializeOutputFile() error {
+	if pr.outputFile == "" {
+		return nil
+	}
+	if !strings.HasSuffix(pr.outputFile, ".json") && !strings.HasSuffix(pr.outputFile, ".yaml") {
+		return fmt.Errorf("output file must have be *.json or *.yaml")
+	}
+	if strings.HasSuffix(pr.outputFile, ".json") && pr.outputFormat != "json" {
+		return fmt.Errorf("output format is yaml, but output file is *.json")
+	}
+	if strings.HasSuffix(pr.outputFile, ".yaml") && pr.outputFormat != "yaml" {
+		return fmt.Errorf("output format is json, but output file is *.yaml")
+	}
```

Comment:
I don't think we need such an enforcement/validation. I'd give users the possibility to choose the filename they prefer, even without the correct extension.

Reply:
I just want to prevent confusion, maybe I can change it to output a warning instead of an error.